src/main/groovy/com/netflix/gradle/plugins/rpm/RpmCopyAction.groovy

```diff
@@ -121,9 +121,9 @@ class RpmCopyAction extends AbstractPackagingCopyAction<Rpm> {
         }
         // Have to take booleans specially, since they would fail an elvis operator if set to false
         def specCreateDirectoryEntry = lookup(specToLookAt, 'createDirectoryEntry')
-        boolean createDirectoryEntry = specCreateDirectoryEntry ?: task.createDirectoryEntry
+        boolean createDirectoryEntry = ( specCreateDirectoryEntry == null ) ? task.createDirectoryEntry : specCreateDirectoryEntry
```

Comment:
Also there was a comment in the original code that stated that an elvis operator would fail if the boolean was false. [https://github.com/nebula-plugins/gradle-ospackage-plugin/commit/020528c4bed7e2da605391af965062385be920d6#diff-20fa12fca41ada43d0d80aa39ec46540R110](url)